lisa/sut_orchestrator/cloud_hypervisor/platform.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import List, Type
+
+import libvirt  # type: ignore
+
+from lisa.environment import Environment
+from lisa.feature import Feature
+from lisa.node import Node
+from lisa.sut_orchestrator.libvirt.context import NodeContext, get_node_context
+from lisa.sut_orchestrator.libvirt.platform import BaseLibvirtPlatform
+from lisa.tools import QemuImg
+from lisa.util.logger import Logger
+
+from .. import CLOUD_HYPERVISOR
+
+
+class CloudHypervisorPlatform(BaseLibvirtPlatform):
+    @classmethod
+    def type_name(cls) -> str:
+        return CLOUD_HYPERVISOR
+
+    @classmethod
+    def supported_features(cls) -> List[Type[Feature]]:
+        return BaseLibvirtPlatform._supported_features
+
+    def hypervisor_name(self) -> str:
```

Comment:
how about name it as `protocol_name` or `schema_name`? It explicitly explains where it's used.

Reply:
Will rename this

Reply:
Renamed it.